README.md

```diff
@@ -12,75 +12,74 @@ Please note that this tool is currently in Preview, and need your feedback for i
-
-## Current Limitations
-- Some file system APIs have not been implemented: readlink, symlink, link, chmod, chown, fsync, lock and extended attribute calls.
-- Not optimized for updating an existing file. blobfuse downloads the entire file to local cache to be able to modify and update the file
-- Does not support SAS yet
-- High latency compared to local filesystems. Further you are from the Azure region, higher the latency is.
-- Currently does not implement data integrity checks. Always have your data backed up before using blobfuse. Use Https as a data integrity mechanism over the wire.
-- Because Azure Block Blobs do not have file and directory semantics, certain directory operations may not behave entirely as expected. For example, deleting the last file in a directory may also delete the directory.
-
 ## Installation
-
-You can install blobfuse from the Linux Software Repository for Microsoft products. The process is explained in the [blobfuse installation](https://github.com/Azure/azure-storage-fuse/wiki/Installation) page. Alternatively, you can clone this repository, install the dependencies (libfuse, libcurl and GnuTLS) and build from source code. See details in the [wiki](https://github.com/Azure/azure-storage-fuse/wiki/Installation#build-from-source).
+You can install blobfuse from the Linux Software Repository for Microsoft products. The process is explained in the [blobfuse installation](https://github.com/Azure/azure-storage-fuse/wiki/Installation) page. Alternatively, you can clone this repository, install the dependencies (fuse, libcurl and GnuTLS) and build from source code. See details in the [wiki](https://github.com/Azure/azure-storage-fuse/wiki/Installation#build-from-source).
 
 ## Usage
 
 ### Mounting
-Once you have built blobfuse, configure your account credentials in the template provided in blobfuse folder (connection.cfg).
+Once you have installed blobfuse, configure your account credentials either in the template provided in blobfuse folder (connection.cfg), or in the environment variables. For brevity, let's use the environment variables:
 
-Here is the format for connection.cfg:
 ```
-accountName <account-name-here> 
-accountKey <account-key-here> 
-containerName <container-name-here>
+export AZURE_STORAGE_ACCOUNT=myaccountname
+export AZURE_STORAGE_ACCESS_KEY=myaccountkey
 ```
 
-Alternatively provide the account name and key in the environment variables AZURE_STORAGE_ACCOUNT and AZURE_STORAGE_ACCESS_KEY, and set --container-name while mounting.
-
-By default, blobfuse will use the ephemeral disks in Azure VMs as the local cache (/mnt/blobfusetmp). Please make sure that your user has write access to this location. If not, create and `chown` to your user.
+By default, blobfuse will use the ephemeral disks in Azure VMs as the local cache (/mnt/blobfusetmp). Please make sure that your user has write access to this location. If not, create and `chown` to your user. For best performance, consider using a ramdisk.
```

Comment:
I don't think "by default" is correct here - if you don't specify, blobfuse fails, it doesn't pick a default